tox.ini

```
@@ -6,8 +6,7 @@ deps = ansible27: ansible<2.8
        ansible28: ansible<2.9
        ansible29: ansible<2.10
        ansible-base: ansible-base
-       py36: ansible-core
-       py37: ansible-core
+       py3: ansible-core
        py38: ansible-core
```

Would this work?
```suggestion
       py{,3,38}: ansible-core
```

It's a common pattern to run `tox -e python` or `tox -e py` which corresponds to just the `py` factor.

---

I've not encountered those patterns, tbh, and I honestly don't know if that expression works, but `py3` associates the need for a python 3.x version, where as the other two (`python` and `py`) do not. I think we should stick with `py3` here.

---

I think that should be an alias. A popular way of setting up the CI is usually choosing the Python interpreter on the job matrix level and then running `tox -e py` or even having `TOXENV=python tox`. w/o needing to calculate the exact python factor to put as an argument for `-e`.